plugins/woocommerce/includes/class-wc-ajax.php

```diff
@@ -3120,11 +3138,19 @@ public static function shipping_zone_methods_save_changes() {
 
 		$zone_id = wc_clean( wp_unslash( $_POST['zone_id'] ) );
 		$zone    = new WC_Shipping_Zone( $zone_id );
+		if ( '' === $zone_id ) {
```

Comment:
Same question as above, why does this have to be an empty string?

Reply:
Same as above: there is a scenario where the shipping zone is created here. In that scenario, I saw that the shipping zone id is blank. The scenario happens when you add a shipping method without first saving the shipping zone.

Reply:
Aaah ok, could we add an inline comment mentioning this?